ledger/participant-integration-api/src/main/scala/platform/store/backend/common/ContractStorageBackendTemplate.scala

```diff
@@ -94,30 +82,27 @@ class ContractStorageBackendTemplate(queryStrategy: QueryStrategy) extends Contr
   // TODO append-only: consider pulling up traversal logic to upper layer
   override def maximumLedgerTime(
       ids: Set[ContractId]
-  )(connection: Connection): Try[Option[Timestamp]] = {
-    if (ids.isEmpty) {
-      Failure(emptyContractIds)
```

Comment:
Are you saying that an empty list will not happen anymore?

Reply:
that's right. not in the types though :(